app/views/admin/policy_configurations/edit.html.erb

```diff
@@ -1,22 +1,32 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-xl">
-      <span class="govuk-caption-xl">Open or close a service</span>
+      <span class="govuk-caption-xl">Manage services</span>
       <%= policy_service_name(@policy_configuration.policy.routing_name) %>
     </h1>
 
     <div class="govuk-panel govuk-panel--informational">
       <% if @policy_configuration.open_for_submissions? %>
         <h3 class="govuk-panel__title">Service open</h3>
-        <p class="govuk-panel__body">This service is currently open. Users can submit claims.</p>
+        <p class="govuk-panel__body">
+          This service is currently open and accepting claims for the
+          <%= @policy_configuration.current_academic_year%> academic year.
+        </p>
       <% else %>
         <h3 class="govuk-panel__title">Service closed</h3>
-        <p class="govuk-panel__body">This service is currently closed. Users can not submit claims.</p>
+        <p class="govuk-panel__body">
+          This service is currently closed. Users can not submit claims.
+        </p>
       <% end %>
     </div>
 
     <%= form_with model: [:admin, @policy_configuration] do |f| %>
 
+      <div class="govuk-form-group">
+        <%= f.label :current_academic_year, "Accepting claims for academic year", class: "govuk-label" %>
```

Comment:
I wonder is `Accepting claims` could cause confusion? It would be possible to be accepting claims for 2020/2021 but have the service closed.

Perhaps something like claim window would be more appropriate?

Reply:
We need to be careful here, there is a technical and semantic difference between the academic year and the claim window. The value here represents the academic year, which runs September to July, whilst the claim window runs September to end of Feb. I think we should go with how it is now and see if service operators find it confusing. When I showed it to Phil last week, he was happy with it.